examples/express/README.md

```diff
@@ -0,0 +1,62 @@
+# ExpressJS example sgID app
+
+The example application code is in `index.ts`. You can copy this code to bootstrap your sgID client application, and run this app locally to understand how this SDK helps you interact with the sgID server.
+
+## Running this example app locally
+
+### Prerequisites
+
+Register a new client at the [sgID developer portal](https://developer.id.gov.sg). For detailed instructions on how to register, check out our [documentation](https://jie-hao-kwa.gitbook.io/sgid-v2-beta/~/changes/75HA2ZSHqQMuHPNECzfs/introduction/getting-started/registration). Feel free to register a test client; there is no limit to the number of clients you can create.
```

Comment:
we'll need to update the docs link haha

Reply:
fixed in [930c29](https://github.com/opengovsg/sgid-client/pull/50/commits/930c29082d92ec6cbd0d6d48044c24ecce1ac3e7)